src/ReactWrapper.jsx

```diff
@@ -885,12 +885,23 @@ class ReactWrapper {
   }
 
   /**
-   * Returns true if the current wrapper has no nodes. False otherwise.
+   * Delegates to exists()
    *
    * @returns {boolean}
    */
   isEmpty() {
-    return this.length === 0;
+    // eslint-disable-next-line no-console
+    console.warn('Deprecated method isEmpty() called, use exists() instead.');
```

Comment:
>I'm not sure how we'd "only log once" - store a global?

React uses a scoped flag to only log warnings once, but I think in our case it makes the most sense to just log every time. Test output can be rather verbose, making a single warning potentially hard to distinguish. `isEmpty` isn't likely used with extreme frequency in the average test suite either, so in most cases we're probably not going to be outputting too much noise.

Also, in regards to versioning: semver specifies that deprecation warnings [should go in minor releases](https://github.com/mojombo/semver/blob/master/semver.md#how-should-i-handle-deprecating-functionality) so I'm 👍 with that.